lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -40,12 +40,14 @@ def check_config(self, client, server_config, host, my_headers={}):
         uri = urljoin(host, uri_prefix)
         auth_realm = server_config.get("authentication", "realm")
         auth_issuer = server_config.get("authentication", "server_url")
+        auth_client = server_config.get("authentication", "client")
+        auth_secret = server_config.get("authentication", "secret")
         expected_results = {
             "authentication": {
                 "realm": auth_realm,
-                "client": server_config.get("authentication", "client"),
+                "client": auth_client,
                 "issuer": auth_issuer,
-                "secret": "",
+                "secret": auth_secret,
             },
```

Comment:
As I mentioned above, I believe we could replace this with
```
        expected_results = {
            "authentication": server_config["authentication"],
```
if we wanted to.

Reply:
Tracked in #3143.